src/metabase/channel/impl/email.clj

```diff
@@ -306,33 +306,35 @@
-                                                  :path    "metabase/channel/email/data_editing_row_update.hbs"}}
-   :event/data-editing-row-delete {:channel_type :channel/email
-                                   :details      {:type    :email/handlebars-resource
-                                                  :subject "Table {{payload.event_info.table.name}} has a row deleted"
-                                                  :path    "metabase/channel/email/data_editing_row_delete.hbs"}}})
+(def ^:private action->template
+  {:row/create {:channel_type :channel/email
+                :details      {:type    :email/handlebars-resource
+                               :subject "Table {{payload.event_info.table.name}} has a new row"
+                               :path    "metabase/channel/email/data_editing_row_create.hbs"}}
+   :row/update {:channel_type :channel/email
+                :details      {:type    :email/handlebars-resource
+                               :subject "Table {{payload.event_info.table.name}} has been updated"
+                               :path    "metabase/channel/email/data_editing_row_update.hbs"}}
+   :row/delete {:channel_type :channel/email
+                :details      {:type    :email/handlebars-resource
+                               :subject "Table {{payload.event_info.table.name}} has a row deleted"
+                               :path    "metabase/channel/email/data_editing_row_delete.hbs"}}})
 
 (mu/defmethod channel/render-notification
   [:channel/email :notification/system-event]
   [_channel-type
    notification-payload #_:- #_notification/NotificationPayload
-   template             :- ::models.channel/ChannelTemplate
+   template             :- [:maybe ::models.channel/ChannelTemplate]
    recipients           :- [:sequential ::models.notification/NotificationRecipient]]
   (let [event-topic (get-in notification-payload [:payload :event_topic])
-        template    (or
-                     template
-                     (get event-name->template event-topic))]
-    (assert template (str "No template found for event " event-topic))
-    [(construct-email (channel.params/substitute-params (-> template :details :subject) notification-payload)
-                      (notification-recipients->emails recipients notification-payload)
-                      [{:type    "text/html; charset=utf-8"
-                        :content (channel.template/render-template template notification-payload)}]
-                      (-> template :details :recipient-type keyword))]))
+        template    (or template
+                        (when (= :event/action.success event-topic)
+                          (get action->template (get-in notification-payload [:payload :event_info :action]))))]
+    #_(assert template (str "No template found for event " event-topic))
+    (if-not template
+      []
```

Comment:
Probably want a better, generic way to opt only specific action's success events triggering any channels.